apps/backoffice-v2/src/@types/react-table.d.ts

```
@@ -0,0 +1,11 @@
+import '@tanstack/react-table';
+
+declare module '@tanstack/react-table' {
+  import { RowData } from '@tanstack/react-table';
+
+  interface ColumnMeta<TData extends RowData, TValue> {
+    useWrapper?: boolean;
+  }
+}
+
+export {};
```

Biome (1.9.4)

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)